code/test/security.js

```diff
@@ -240,12 +445,8 @@ contract('ONEWallet', (accounts) => {
   // Authentication: from function authenticate in reveal.sol
   // if innerCores are empty, this operation (in this case) is doomed to fail. This is intended. Client should warn the user not to lower the limit too much if the wallet has no innerCores (use Extend to set first innerCores). Client should also advise the user the use Recovery feature to get their assets out, if they are stuck with very low limit and do not want to wait to double them each spendInterval.
   it('SE-COMPLEX-24-25 SPENDING LIMIT RULES: must be able to update spending limit according to the rules', async () => {
-    // create wallets and token contracts used througout the tests
```

Comment:
need to double check on this test because spendInterval is substantially reduced compared to original tests

Reply:
currently it runs and passes